.github/workflows/prod_test.yml

```diff
@@ -0,0 +1,37 @@
+name: End-to-end Production Testing (Nightly)
+on:
+  schedule:
+    # Run every morning Pacific Time.
+    # Random hour and minute to avoid creating excess traffic during popular times.
+    # Because the test takes a long time (> 30 min) to run, it is configured to run only once a day.
+    - cron:  '17 17 * * *'
```

Comment:
can you submit a revision of this review that runs on pull request so that we can see *this* PR run the tests, and once that is succeeding we can remove that check before merging